app/test_main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import unittest
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+class TestCanAccessGooglePage(unittest.TestCase):
+    @mock.patch("requests.get")
+    @mock.patch("datetime.datetime")
```

Comment:
i know it is quite hard to mock datetime function, but the task was to mock other funcs
![image](https://github.com/mate-academy/py-can-access-google-page/assets/103201010/c6ef2a47-7793-4040-a090-edb15649e30e)
it seems to me, that task would be much easier for u, if u choose this funcs to mock)
after mocking this funcs, u could test the work of function changing their return values (u could use parametrize for this)

Reply:
Well, in this case if I understood you right it's much easier to do than what I was trying to do :)
I'll try to fix it after the morning lesson